src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -437,6 +437,59 @@ end
     return SVector(flux_inner[1], flux_inner[2], flux_inner[3], flux_inner[4])
 end
 
+# Computes the mirror velocity across a symmetry plane which enforces
+# a tangential velocity that is aligned with the symmetry plane, i.e.,
+# which is normal to the `normal_direction`.
```

Comment:
We use the same technique to impose hyperbolic reflective wall boundary conditions - could you xref that code? 

Also, it might be good to make a note that the slip wall BCs assume you are also using reflective slip wall BCs for the hyperbolic BCs.